layers/+completion/auto-completion/packages.el

```diff
@@ -150,59 +149,10 @@
        (unless (eq auto-completion-enable-help-tooltip 'manual)
          (company-quickhelp-mode))))))
 
-(defun auto-completion/init-all-the-icons ()
```

Comment:
*Should* this be moved? I can also just change this to a pre-init hook and move ownership.

Reply:
Hmm we need to make sure that company box continues to work for users not using all-the-icons everywhere. 

Not sure if this package works at all at the moment but nevertheless the necessary config for it must be applied even when all-the-icons-layer is not loaded.

Lets wait with this point until we have a confirmation that company-box works at all if it does not we can just keep it as it is.

Reply:
Company box should still work, this is just the part that configures company-box to use all-the-icons.

Reply:
This should be fine then